recipes/jags/meta.yaml

```diff
@@ -0,0 +1,30 @@
+{% set name = "JAGS" %}
+{% set version = "4.3.0" %}
+{% set sha256 = "8ac5dd57982bfd7d5f0ee384499d62f3e0bb35b5f1660feb368545f1186371fc" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://sourceforge.net/projects/mcmc-jags/files/JAGS/4.x/Source/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - toolchain
+    - gcc
+    - lapack
+
+about:
+    home: http://mcmc-jags.sourceforge.net
+    license: GPL v2
```

Comment:
Should be `GPL-2.0`. Also add `license_family: GPL` and `license_file: COPYING`.